test/sql/drop_rename_hypertable.sql

```diff
@@ -16,7 +16,7 @@ ALTER TABLE "renametable" RENAME TO "newname";
 SELECT * FROM "newname";
 
 SELECT * FROM _timescaledb_catalog.hypertable;
-DROP TABLE "two_Partitions";
+DROP TABLE "two_Partitions" CASCADE;
```

Comment:
Now hypertables directly inherit chunks. This is actually more correct now imho.